pkg/controller/status/status.go

```diff
@@ -181,6 +188,11 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 				disabledReason = summary.Reason
 				disabledMessage = summary.Message
 			}
+		} else if summary.Operation == controllerstatus.DownloadingReport {
+			degradingFailure = false
+			klog.V(4).Info("Failed to download Insights report")
```

Comment:
I will try to think about some other/better message here....on the other hand it's pretty clear from the previous log messages that the download failed....